homeassistant/components/insteon/utils.py

```diff
@@ -133,6 +170,21 @@ def _send_load_aldb_signal(entity_id, reload):
         signal = f"{entity_id}_{SIGNAL_LOAD_ALDB}"
         dispatcher_send(hass, signal, reload)
```

Comment:
Sync or async context?

Reply:
It is triggered by a service call, which I assume is sync, and the listener is an async method. It works as is but not sure which context it is in.

Reply:
Nope. Definitely called from within the event loop so changing it to async_dispatcher_send, also just including it in the calling method to make it more obvious it is in the async context.